cellmlmanip/rdf.py

```diff
@@ -63,10 +63,24 @@ def create_rdf_node(self, node_content=None, fragment_id=None):
 '''
 
 
-def create_rdf_node(namespace_uri, local_name):
-    """Creates and returns an RDF node ``{namespace_uri}local_name``."""
-    # Ensure namespace prefix can be appended to
-    if namespace_uri[-1] not in ['#', '/']:
-        namespace_uri = namespace_uri + '#'
-
-    return rdflib.Namespace(namespace_uri)[local_name]
+def create_rdf_node(node_content):
+    """Creates and returns an RDF node
+
+    :param symbol: node_content EITHER a string or a tuple (namespace_uri, local_name)"""
+    if node_content is None:
+        return None
+    elif isinstance(node_content, rdflib.Literal) or isinstance(node_content, rdflib.BNode) or \
```

Comment:
Just `isinstance(node_content, rdflib.term.Node):` should work.

Reply:
Aah that's what I was looking for